### PR TITLE
Feat: Canister Home Messaging

### PR DESCRIPTION
**Pull request by lmuntanerDfinity**
# Motivation

Add content in Canisters Home

# Changes

* Add messages for Canisters Home.
* Increase margin in messages in Neurons Home

### CODE REVIEW

**Comment by lmuntanerDfinity** [comment]
@mstrasinskis @peterpeterparker a super simple one.

Here are the screenshots:

Screenshot 2022-02-18 at 13 50 09

Screenshot 2022-02-18 at 13 50 16

File: frontend/svelte/src/routes/Canisters.svelte [hunk 1]
```diff
@@ -19,6 +20,15 @@
...
     <section>
       <h1>{$i18n.canisters.title}</h1>
+      <p>{$i18n.canisters.text}</p>
+      <ul>
+        <li>{$i18n.canisters.step1}</li>
+        <li>{$i18n.canisters.step2}</li>
+        <li>{$i18n.canisters.step3}</li>
+      </ul>
+      <p>
+        {$i18n.canisters.principal_is} "{$authStore.principal}"
```
Comment (peterpeterparker) [inline frontend/svelte/src/routes/Canisters.svelte:30]: `"{$authStore.principal}"` is automatically casted to string?

in `Neurons.svelte` we explicitly use `principal?.toText()`

Comment (lmuntanerDfinity) [reply frontend/svelte/src/routes/Canisters.svelte]: Good point, I didn't see it. It was working but it's better to change it.

**Comment by lmuntanerDfinity** [comment]
@peterpeterparker ready for another review

**Review by peterpeterparker — CHANGES_REQUESTED** [review]
Cool, looks good.

Just one thing, can you add a test?

Probably worth testing that the `principal.toText` is rendered

**Comment by lmuntanerDfinity** [comment]
@peterpeterparker @mstrasinskis ready for review with a test.

Interesting to note that even though I didn't explicitly use the `subscribe` method, instead used the `$` to access the store value, I still had to mock the subscribe method. Otherwise, the value in the store was `undefined`.

**Review by peterpeterparker — APPROVED** [review]
Thanks for the PR

**Pull request merged**